types/react-stonecutter/index.d.ts

```diff
@@ -0,0 +1,200 @@
+// Type definitions for react-stonecutter 0.3
+// Project: https://github.com/dantrain/react-stonecutter
+// Definitions by:  jzipfler <https://github.com/jzipfler>
+//                  Iswenzz <https://github.com/Iswenzz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export type Coordinates = [number, number];
+export interface Layout {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-stonecutter/index.d.ts;
  Line: 11;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.